CDCgov/NEDSS-Modernization
# Fixed Partial phone number search, Event Search Clear All criteria, Comments field to 2000 Characters, Removed comma after state in Address field

## Description

1. Fixed Partial phone number search
2. Fixed Event Search Clear All criteria
3. Comments field to 2000 Characters
4. Removed comma after state in Address field

## Tickets

* [CNFT1-1674](https://cdc-nbs.atlassian.net/browse/CNFT1-1674)
* [CNFT1-1619](https://cdc-nbs.atlassian.net/browse/CNFT1-1619)
* [CNFT1-1643](https://cdc-nbs.atlassian.net/browse/CNFT1-1643)
* [CNFT1-1683](https://cdc-nbs.atlassian.net/browse/CNFT1-1683)


## Steps to verify

1. CNFT1-1683: In Event Search Search for any Patient Information and click on Clear All button it should clear all fields
2. CNFT1-1674: Able to search for a partial phone number (3 Digits, EX: 571)
3. CNFT1-1619: Removed comma after State in the address field
4. CNFT1-1643:  Comments field should allow up to 2000 characters.


[CNFT1-1674]: https://cdc-nbs.atlassian.net/browse/CNFT1-1674?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CNFT1-1619]: https://cdc-nbs.atlassian.net/browse/CNFT1-1619?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CNFT1-1643]: https://cdc-nbs.atlassian.net/browse/CNFT1-1643?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CNFT1-1683]: https://cdc-nbs.atlassian.net/browse/CNFT1-1683?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
